packages/core/useStorageAsync/index.ts

```diff
@@ -94,7 +100,11 @@ export function useStorageAsync<T extends(string | number | boolean | object | n
     }
   }
 
-  read()
+  read().then(() => {
+    if (onLoad) {
+      onLoad(data.value)
+    }
```

Comment:
```suggestion
      onLoad?.(data.value)
```
I think this is a bit more consistent for example when someone accidentally passes a boolean. So `true` and `false` will both result in an error.